src/coreclr/nativeaot/Runtime.Base/src/System/Runtime/MethodTable.Runtime.cs

```diff
@@ -98,6 +98,14 @@ internal static bool BothSimpleCasting(MethodTable* pThis, MethodTable* pOther)
             return ((pThis->_uFlags | pOther->_uFlags) & (uint)EETypeFlags.ComplexCastingMask) == 0;
         }
 
+        internal static bool FastEquals(MethodTable* mt1, MethodTable* mt2)
```

Comment:
I would give this a better name, like `static bool AreSameType(MethodTable* mt1, MethodTable* mt2)`.

Reply:
`AreSameType` sounds better. No need to explain why no equivalence checks on CoreClr side.